src/components/PopupWithConfirm.js

```diff
@@ -27,9 +27,9 @@ export default class PopupWithConfirm extends Popup {
 
```

Comment:
```js

  close() {
    super.close();
    this._popupForm.removeEventListener("submit", this._handleFormSubmit);
  }
```

Please, delete method `close` from `PopupWithConfirm` because you don't need it: you don't need to remove the listener which you don't add